### PR TITLE
ADD OPTIM

### PR DESCRIPTION
**Pull request by zygomeb**
Honestly just looked at how the others did it lmk if there's anything else

### CODE REVIEW

File: src/tokens/optim.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+import { getAmountInAddresses, getBlockFrostInstance } from "../utils";
+
+const OPTIM = "e52964af4fffdb54504859875b1827b60ba679074996156461143dc1";
```
Comment (TheTapHouse) [inline src/tokens/optim.ts:4]: Update this to policy + name

`const OPTIM = "e52964af4fffdb54504859875b1827b60ba679074996156461143dc14f5054494d"`

**Review by TheTapHouse — CHANGES_REQUESTED** [review]
Also import fetcher into `index.ts`

**Comment by zygomeb** [comment]
okay a formatter fail I hate those let's see

**Comment by zygomeb** [comment]
@TheTapHouse seems good to go now

**Review by TheTapHouse — APPROVED** [review]

**Review by shadowkora — APPROVED** [review]

**Pull request merged**